parm/config/gfs/config.atmanl

```diff
@@ -5,12 +5,16 @@
 
 echo "BEGIN: config.atmanl"
 
-export CASE_ANL=${CASE}
-export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/config/
-export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/atm/obs/lists/gdas_prototype_3d.yaml
-export ATMVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/atm/variational/3dvar_drpcg.yaml
+if [[ ${DOHYBVAR} = "YES" ]]; then
+    # shellcheck disable=SC2153
+    export CASE_ANL=${CASE_ENS}
+else
+    export CASE_ANL=${CASE}
+fi
+export OBS_LIST="${HOMEgfs}/parm/gdas/atm/obs/lists/gdas_prototype_3d.yaml.j2"
+export ATMVARYAML="${HOMEgfs}/parm/gdas/atm/variational/3dvar_drpcg.yaml.j2"
```

Comment:
```suggestion
export JEDIYAML="${HOMEgfs}/parm/gdas/atm/variational/3dvar_drpcg.yaml.j2"
```
Can we name this `JEDIYAML` instead of `ATMVARYAML`?  This will allow us to render this yaml once without having to render the yamls for observations, biases, and finally the DA.

Reply:
Change committed at ed042392.